ember-launch-darkly/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "ember-launch-darkly",
+  "version": "2.2.0",
+  "description": "A modern Ember addon to wrap the Launch Darkly service",
+  "keywords": [
+    "ember-addon"
+  ],
+  "repository": "https://github.com/adopted-ember-addons/ember-launch-darkly",
+  "license": "MIT",
+  "author": "The Ember Launch Darkly Team",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "files": [
+    "addon-main.js",
+    "dist"
+  ],
+  "scripts": {
+    "build": "rollup --config",
+    "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
+    "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
+    "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
+    "lint:js": "eslint . --cache",
+    "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
+    "lint:js:fix": "eslint . --fix",
+    "start": "rollup --config --watch",
+    "test": "echo 'A v2 addon does not have tests, run tests in test-app'",
+    "prepack": "rollup --config"
+  },
+  "dependencies": {
+    "@embroider/addon-shim": "^1.0.0",
+    "launchdarkly-js-client-sdk": "^2.24.2",
+    "tracked-maps-and-sets": "^3.0.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.21.3",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
+    "@embroider/addon-dev": "^3.0.0",
+    "@rollup/plugin-babel": "^6.0.3",
+    "babel-eslint": "^10.1.0",
+    "concurrently": "^7.6.0",
+    "ember-source": "4.11.0",
```

Comment:
Do you think we can add it as a peer dep? Example: https://github.com/adopted-ember-addons/ember-launch-darkly/blob/master/package.json#L77